test/elixir_analyzer/exercise_test/common_checks/private_helper_functions_test.exs

```diff
@@ -0,0 +1,452 @@
+defmodule ElixirAnalyzer.ExerciseTest.CommonChecks.PrivateHelperFunctionsTest do
+  use ExUnit.Case
+
+  alias ElixirAnalyzer.ExerciseTest.CommonChecks.PrivateHelperFunctions
+  alias ElixirAnalyzer.Comment
+  alias ElixirAnalyzer.Constants
+
+  @pacman_stub "elixir/exercises/concept/pacman-rules/lib/rules.ex"
+               |> File.read!()
+               |> Code.string_to_quoted()
+
+  @sqrt_stub "elixir/exercises/practice/square-root/lib/square_root.ex"
+             |> File.read!()
+             |> Code.string_to_quoted()
+
+  @comment %Comment{
+    type: :informative,
+    comment: Constants.solution_private_helper_functions(),
+    name: Constants.solution_private_helper_functions()
+  }
+
+  describe "concept exercice with pacman-rules" do
```

Comment:
Ok, that's enough. Tell me which editor you're using and I will search for a built-in English dictionary for it 😂

Reply:
OMG!!! I use vim, which can use `:set spell`. It is now always turned on.